plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -64,6 +68,12 @@ function assertSpan(
     attributes[SemanticAttributes.DB_STATEMENT] = query;
   }
 
+  if (customAttributes) {
+    for (const [k, v] of Object.entries(customAttributes)) {
+      attributes[k] = v;
+    }
+  }
```

Comment:
nit: You can use the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) and merge this with above definition like this:

```js
const attributes: SpanAttributes = {
    [SemanticAttributes.DB_SYSTEM]: DbSystemValues.CASSANDRA,
    [SemanticAttributes.DB_USER]: 'cassandra',
    ...(customAttributes ?? {}),
}
```